Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:jessie
```

Comment:
do we have an agreement on which base image we should use.
alpine should be enough?

Reply:
I used alpine, but the `date` command in alpine behaves different from what I was expecting. Since this is not a service, but a one-off tool, I don't give a lot about image size or other factors.

Reply:
ok fine